frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/configure-size.tsx

```diff
@@ -14,7 +14,7 @@ export const configureSizeModal = ({
     defaultValue: specValue || 0,
     title: `Modify ${specDescriptor.displayName}`,
     message: specDescriptor.description,
-    path: `/spec/${specDescriptor.path}`,
+    path: `/spec/${specDescriptor.path.replace(/\]/g, '').replace(/[.[]/g, '/')}`,
```

Comment:
I'd suggest using `_.toPath` and then building up the patch path from the array. Something like (untested)

```suggestion
    path: ['spec', _.toPath(specDescriptor.path)].join('/'),
```

Reply:
@spadgett fixed